e2etest/pkg/config/frrprovider.go

```diff
@@ -0,0 +1,118 @@
+// SPDX-License-Identifier:Apache-2.0
+
+package config
+
+import (
+	"fmt"
+
+	"go.universe.tf/e2etest/pkg/executor"
+	"go.universe.tf/e2etest/pkg/metallb"
+	corev1 "k8s.io/api/core/v1"
+	clientset "k8s.io/client-go/kubernetes"
+	"k8s.io/client-go/rest"
+)
+
+type FRRProvider interface {
```

Comment:
how about an frrprovider package (and renaming everything accordingly?)

Reply:
done (moved to `e2etest/pkg/frr/provider`)